.cargo/config.toml

```diff
@@ -14,3 +14,22 @@ rustflags = [
     # in line with Linux (whereas default for Windows is 1MB)
     "-Clink-arg=/STACK:10000000",
 ]
+
+
+[build]
+target = "valida-unknown-baremetal-gnu"
+
+[target.valida-unknown-baremetal-gnu]
+runner = "/valida-toolchain/valida-runner.sh"
+linker = "/valida-toolchain/bin/ld.lld"
+rustflags = [
+    "-C", "link-arg=/valida-toolchain/ValidaEntryPoint.o",
```

Comment:
Missing `io.o`. That's not reported because of `--noinhibit-exec` :dancing_men:

Reply:
Perhaps it does not affect this PR yet if it's based on older toolchain but on the most recent one it will